app/src/main/java/org/fossasia/susi/ai/skills/SkillsActivity.kt

```diff
@@ -61,15 +62,25 @@ class SkillsActivity : AppCompatActivity(), SkillFragmentCallback {
 
         settingsPresenter = SettingsPresenter(this)
         val skillFragment = SkillListingFragment()
-        //skills = skillFragment.skills
-        supportFragmentManager.beginTransaction()
-                .add(R.id.fragment_container, skillFragment, TAG_SKILLS_FRAGMENT)
-                .addToBackStack(TAG_SKILLS_FRAGMENT)
-                .commit()
+        val privacyFragment = PrivacyFragment()
+        val bundle = intent.extras
+        val intentValue = bundle?.getBoolean(Constant.SIGN_UP_TO_PRIVACY)
```

Comment:
What intentValue. I don't understand the name of this variable

Reply:
Sir, this is a situation where we are trying to open a fragment that is contained inside another activity. The other activity doesn't have the default fragment as the one that we require. And we know that we cannot open the fragment of another activity directly until it is the default fragment of that activity. So here along with the intent I am passing the parameter that I have called the activity from Signup page. It should open the Privacy fragment instead of SkillsListingFragment(the default fragment of this activity).

Reply:
I'm talking about using a sensible variable name

Reply:
Sir, I am sorry for that. I am changing that variable name to an appropriate one.